ui/components/Navbar/Navbar.js

```diff
@@ -369,7 +369,7 @@ const Navbar = ({
         onClose={() => setIsDrawerOpen(false)}
         className={isDrawerCollapsed ? classes.sidebarCollapsed : classes.sidebarExpanded}
         classes={{ paper: isDrawerCollapsed ? classes.sidebarCollapsed : classes.sidebarExpanded }}
-        style={{ width: "inherit" }}
+        sx={{ zIndex: theme.zIndex.drawer + 10, position: "relative", width: "inherit" }}
```

Comment:
Can you avoid inline css here @abhinavkrin

Reply:
Done!